packages/cron/src/jobs/storage.js

```diff
@@ -0,0 +1,74 @@
+import debug from 'debug'
+import { EmailService } from '../lib/email.js'
+import { EMAIL_TYPE } from '@web3-storage/db'
+
+const log = debug('storage:checkStorageUsed')
+let emailService
+
+/**
+ * Get users with storage quota usage in percentage range and email them as
+ * appropriate when approaching their storage quota limit.
+ * @param {{
+ *   db: import('@web3-storage/db').DBClient
+ * }} config
+ */
+export async function checkStorageUsed ({ db }) {
+  if (!log.enabled) {
+    console.log('ℹ️ Enable logging by setting DEBUG=storage:checkStorageUsed')
+  }
+
+  log('🗄 Checking users storage quotas')
+
+  let users
+  emailService = new EmailService({ db })
+
+  // Send email every day
+  users = await db.getUsersByStorageUsed({
+    fromPercent: 100
+  })
+  await emailService.sendEmails({
+    users,
+    email: EMAIL_TYPE.Used100PercentStorage,
+    numberOfDays: 1
+  })
+
+  users = await db.getUsersByStorageUsed({
+    fromPercent: 90,
+    toPercent: 100
+  })
+  await emailService.sendEmails({
+    users,
+    email: EMAIL_TYPE.Used90PercentStorage,
+    numberOfDays: 1
+  })
+
+  // Send email every 7 days
+  users = await db.getUsersByStorageUsed({
+    fromPercent: 85,
+    toPercent: 90
+  })
+  await emailService.sendEmails({
+    users,
+    email: EMAIL_TYPE.Used90PercentStorage
```

Comment:
Can you please double check all the email types in this file?

Reply:
Good catch, have corrected and added to the test.